crates/locutus-node/src/operations/subscribe.rs

```diff
@@ -385,6 +446,8 @@ mod messages {
             key: ContractKey,
             target: PeerKeyLocation,
             subscriber: PeerKeyLocation,
+            retries: usize,
```

Comment:
For consistency across the application let's call this `htl` (stands for `hops to live`).